src/index.ts

```diff
@@ -450,7 +453,10 @@ export const useForm = <Values extends Record<string, any>, ErrorMessage = strin
       const wasEditing = formStatus.current === "editing";
       formStatus.current = "submitting";
 
-      const names: Name[] = Object.keys(mounteds.current);
+      const names = Object.keys(mounteds.current).reduce<Name[]>((acc, name) => {
```

Comment:
maybe here you can use just `.filter` instead of `.reduce`

Reply:
Good catch 👍